content/blog/learning-to-manage-ilo-7-with-restful-interface-tool-ilorest-6-0.md

```diff
@@ -0,0 +1,772 @@
+	mso-para-margin-top:0in;
+	mso-para-margin-right:0in;
+	mso-para-margin-bottom:8.0pt;
+	mso-para-margin-left:0in;
+	line-height:107%;
+	mso-pagination:widow-orphan;
+	font-size:11.0pt;
+	font-family:"Calibri",sans-serif;
+	mso-ascii-font-family:Calibri;
+	mso-ascii-theme-font:minor-latin;
+	mso-hansi-font-family:Calibri;
+	mso-hansi-theme-font:minor-latin;
+	mso-bidi-font-family:"Times New Roman";
+	mso-bidi-theme-font:minor-bidi;
+	mso-ansi-language:EN-US;
+	mso-fareast-language:EN-US;}
+</style>
+<!\\\[endif]-->
+
+<!--StartFragment-->
+
+## Introduction
+
+We know you love **HPE servers** for their **security** and **ease of management**. With the release of **HPE iLO 7** on **Gen12 servers**, HPE has taken security to the next level. Unlike previous generations where **Production** was the default <a href="https://servermanagementportal.ext.hpe.com/docs/redfishservices/ilos/supplementdocuments/securityservice/#ilo-security-state" target="_blank">security state</a>, iLO 7 introduces the **Secure Standard** state by default, enhancing the system security.
+
+Another major shift is in the default login method - **Virtual NIC (<a href="https://servermanagementportal.ext.hpe.com/docs/redfishservices/ilos/supplementdocuments/vnic/#the-ilo-redfish-host-interface-virtual-nic" target="_blank">VNIC</a>)** replaces the **<a href="https://servermanagementportal.ext.hpe.com/docs/etc/glossaryterms/" target="_blank">CHIF</a> interface**, which was used in **iLO 6 and earlier versions**.
+
+In this article, I’ll walk you through **managing HPE iLO 7 using the iLOrest 6.0 tool via in-band access**. We’ll start with **installing <a href="https://github.com/HewlettPackard/python-redfish-utility/releases/latest" target="_blank">iLOrest</a>** on different operating systems and then dive into **logging into iLO 7**. Since **iLO 7 introduces a new <a href="https://servermanagementportal.ext.hpe.com/docs/redfishservices/ilos/supplementdocuments/securityservice/#application-accounts" target="_blank">application account</a> login method**, I’ll also cover how iLOrest 6.0 fully supports this feature.
+
+Let’s get started! 🚀
+
+<!--EndFragment-->
+
+## Installation
+
+### Linux:
+
+On Linux, iLOrest can be installed as an RPM package. If you already have a previous version installed, you can upgrade it using the -Uvh option.
+For a fresh installation, use:
+
+```bash
+rpm -ivh ilorest-6.0.x86_64.rpm
+```
+
+Here is a screenshot of successful installation.
+
+![](/img/rpm_linux.jpg)
+
+**Application account creation during iLOrest installation**
+
+During the RPM installation of iLOrest 6.0, you might notice that the installer prompts for iLO credentials to create an Application account. While this step is optional, HPE strongly recommends creating the Application account during installation itself.
+Why?
+
+•	The Application account provides an additional method for in-band authentication with iLO 7, enhancing security and flexibility.
```

Comment:
Interesting. The formatting of this bullet and the next isn't actually acting like a bullet. The second line doesn't auto indent. Maybe Denis can help with the formatting here?

![image](https://github.com/user-attachments/assets/ae58800c-6abf-4a8b-b2fe-8307003c2286)